Package.swift

```diff
@@ -61,7 +61,7 @@ let package = Package(
         ),
         .target(
             name: "SwiftLintFramework",
-            dependencies: frameworkDependencies
+            dependencies: frameworkDependencies + ["SwiftyTextTable"]
```

Comment:
Sadly, had to add this linker dependency :-(

Reply:
Which I guess was not enough, as CI and `bazel build :swiftlint` are failing with 

```
error: emit-module command failed with exit code 1 (use -v to see invocation)
Source/SwiftLintFramework/Reporters/SummaryReporter.swift:2:8: error: no such module 'SwiftyTextTable'
import SwiftyTextTable
       ^
Source/SwiftLintFramework/Reporters/SummaryReporter.swift:2:8: error: no such module 'SwiftyTextTable'
import SwiftyTextTable
```

Reply:
OK - I think I've figured the bazel linking out